src/components/utils/AdvertisingConfigPropType.js

```diff
@@ -85,9 +85,11 @@ export default PropTypes.shape({
         )
     ),
     slots: PropTypes.arrayOf(AdvertisingSlotConfigPropType),
-    outOfPage: PropTypes.shape({
-        id: PropTypes.string
-    }).isOptional,
+    outOfPageSlots: PropTypes.arrayOf(
+        PropTypes.shape({
+            id: PropTypes.string
+        })
+    ).isOptional,
```

Comment:
to my knowledge, there is no such thing as `isOptional` in prop types. If it doesn't have `isRequired`, it is optional

Reply:
ok, I will change it...